lametro/events.py

```diff
@@ -397,17 +394,5 @@ class LAMetroWebEvent(dict):
 
     @property
     def has_audio(self):
-        return self['Audio'] != 'Not\xa0available'
-
-    @property
```

Comment:
We don't need any of these properties anymore because the way that are now creating the web_event is by visiting the legistar meeting detail page (in python-legistar). If we can't visit it, we wont' create the web_event.